packages/x-charts/package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@mui/x-charts",
-  "version": "6.19.5",
+  "version": "6.19.6",
```

It doesn't look like this package received any changes since the last release. 🙈 
```suggestion
  "version": "6.19.5",
```

---

But then the versions don't match and I find that more confusing.

---

What about `@mui/x-tree-view` and `@mui/x-license` then? 🤔

---

Restored & committed but github seems to lag behind even though the commits have been pushed. To be confirmed.